jurity/recommenders/doubly_robust.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
```

Comment:
I can be persuaded either way, but since both DR and IPS calculate CTR scores, it might be a bit confusing to have them as separate implementations. I'm wondering if you could add a new argument `estimation` to the CTR initializer, where `estimation` is one of `matching`, `IPS`, `DR`? Would that be more confusing on the long run?

Reply:
Oh I like that idea! It would help to clarify how these are alternative ways of estimating CTR

@bkleyn @filip-michalsky what do you think?

Reply:
Yeah I think it makes sense. It would make the link between these methods more explicit. 

Think it would be good to add `estimation` argument to __str__ to allow easy identification of "which" method was used.

Reply:
I pushed an update merging IPS and DR into the CTR metric